### DISCUSSION TITLE
zsh: no such user or named directory with ~ argument

### DISCUSSION THREAD

**Discussion by wangzhankun** [ohmyzsh/ohmyzsh#9857] [Help] [answered]
I want to install lib using cargo install, but error happened:
```bash
$ cargo install cargo-binutils --vers ~0.2
zsh: no such user or named directory: 0.2
```
But when I use bash, the command is right.

**Comment by mcornella** [comment, marked as answer]
This is not a bug but a feature of zsh. Arguments that start with `~` are called `named directories`, so if you do for example `~user` that gets expanded to the user's home directory (`~` gets expanded to your current user's home directory for example). You need to quote that tilde with `\` or putting it inside quotes (`'~0.2'` or `"~0.2"`).

[See more info here](http://zsh.sourceforge.net/Doc/Release/Expansion.html#Filename-Expansion).
